buildkite/bazel-central-registry/bcr_postsubmit.py

```diff
@@ -166,30 +45,9 @@ def sync_bcr_content():
         ["gsutil", "-h", "Cache-Control:no-cache", "-m", "rsync", "-d", "-r", "./modules", BCR_BUCKET + "modules"]
     )
 
-
-def update_last_synced_commit(current_commit):
-    print_expanded_group(":gcloud: Update last synced commit")
-    subprocess.check_output(
-        "echo %s | gsutil -h 'Cache-Control: no-cache' cp - %s" % (current_commit, LAST_SYNCED_COMMIT_URL),
-        shell=True,
-    )
-    eprint("Last synced commit updated to " + current_commit)
-
-
 def main():
-    last_synced_commit = fetch_last_synced_commit()
```

Comment:
I'm guessing that since the registry contents aren't that big, we can just always copy them?

Reply:
Yes, I believe it should last a very long time until we hit any kind of performance problem.